view/frontend/templates/instant/hit.phtml

```diff
@@ -12,7 +12,7 @@ $tierFormatedVar = 'price' . $priceKey . '_tier_formated'
 <script type="text/template" id="instant-hit-template">
     <div class="col-md-4 col-sm-6">
         <div class="result-wrapper" itemprop="itemListElement" itemscope itemtype="http://schema.org/Product">
-            <a itemprop="url" href="{{url}}" class="result" data-objectid="{{objectID}}" data-position="{{__position}}">
+            <a itemprop="url" href="{{url}}" class="result" data-objectid="{{objectID}}" data-position="{{__position}}" data-queryid={{__queryid}}>
```

Comment:
Nit: respecting the original letter case would lead to more predictable data attributes:

- Object ID → `data-object-id`
- Query ID → `data-query-id`